aspnetcore/host-and-deploy/azure-apps/index.md

```diff
@@ -47,6 +47,8 @@ Understand and troubleshoot warnings and errors with ASP.NET Core projects.
 
 ### Platform
 
+The platform architecture (x86/x64) of an App Services app is set in the app's settings in the Azure Portal for apps that are hosted on an A-series compute (Basic) or higher hosting tier. Confirm that the app's publish settings (for example, in the Visual Studio [publish profile (.pubxml)](xref:host-and-deploy/visual-studio-publish-profiles)) match the setting in the app's service configuration in the Azure Portal.
```

Comment:
Good to mention this here. Do we have a screen shot of the portal where this is done? Do we think it'd add value to show a AZ CLI script to set/toggle it?

Reply:
We try to avoid screenshots. UI churn is a *killer* here ... we end up with stale images *very quickly*. Therefore, we try to link out to Azure docs, where the screenshots will probably be updated in a timely fashion. Having said that, you can just say, *'I demand a screenshot YOU GooberRex!'* 👊:smile: ... and of course, I'll take one for this.

WRT AZ CLI script for it, best to link to it over in Azure docs if possible. If there's nothing to link, sure thing ... can you give me the script commands?

Also ... you checked my opening comments ☝️? Any changes based on those questions/concerns?

Reply:
@bradygaster We left this discussion off at the "AZ CLI script" to set the platform architecture in Azure ... is there something that I can cross-link? ... or do you have the critical script commands that I can put here?